Tribler/community/multichain/database.py

```diff
@@ -288,7 +288,11 @@ def check_database(self, database_version):
         assert int(database_version) >= 0
         database_version = int(database_version)
 
-        if database_version < 1:
+        if database_version != LATEST_DB_VERSION:
```

Comment:
I think it's better to check for `database_version < 2` then upgrade to 2. This is in line with all other database upgrade procedures.